lib/doorkeeper/openid_connect.rb

```diff
@@ -48,7 +48,7 @@ def self.signing_key
         else
           OpenSSL::PKey.read(configuration.signing_key)
         end
-      ::JWT::JWK.new(key)
+      ::JWT::JWK.new(key, { kid_generator: JWT::JWK::Thumbprint })
```

Comment:
**Suggestion**: add this change to the [unreleased section of the Changelog file ](https://github.com/doorkeeper-gem/doorkeeper-openid_connect/blob/master/CHANGELOG.md#unreleased) and maybe backfill some info into the `1.8.4` Changelog so that folks upgrading to that version know about the change in `kid` format for that version?

Reply:
+1 @stanhu :point_up:

Reply:
@nbulaj Done!